tests/test_storage_macro_dbc.rs

```diff
@@ -0,0 +1,76 @@
+use common::*;
+mod common;
+
+fn _get_mysql_connection_url() -> String {
+    if std::env::var("MYSQL_DATABASE_URL").is_ok() {
+        std::env::var("MYSQL_DATABASE_URL").unwrap()
+    } else {
+        "mysql://172.24.176.1/test_db?user=nociza&password=password".to_string()
+    }
+}
+
+#[tokio::test]
+async fn test_storage_macro_dbc_mysql(
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let (_ir, _is, cl) = set_up_test_env_single_user().await?;
+
+    cl.create_entry(
+        "storage_macro_test:db:url",
+        _get_mysql_connection_url().as_bytes(),
+    )
+    .await?;
+
+    cl.create_entry(
+        "storage_macro_test:db:create_db",
+        b"CREATE DATABASE IF NOT EXISTS test_db" as &[u8],
+    )
+    .await?;
+
+    // Create a table and insert dummy data
+    cl.create_entry(
+        "storage_macro_test:db:create_table",
+        b"CREATE TABLE IF NOT EXISTS users (name VARCHAR(255), email VARCHAR(255))" as &[u8],
+    )
+    .await?;
+
+    cl.create_entry(
+        "storage_macro_test:db:insert_data",
+        b"INSERT INTO users VALUES ('Alice', 'alice<at>berkeley.edu')" as &[u8],
```

Comment:
let's just use age (instead of email) as an example to simplify, also we can test the use of numbers in this way

Reply:
Done.